src/components/visualizations/chart/tooltip.ts

```diff
@@ -14,12 +14,15 @@ export function formatValueForTooltip(
 }
 
 export function getFormattedValueForTooltip(
+    isDualAxis: boolean,
+    isSecondAxis: boolean,
```

Comment:
I think we can merge `isDualAxis` and `isSecondAxis` to `isDualChartWithRightAxis`

Reply:
updated